src/app/weather/weather-simulation/weather-simulation.component.html

```diff
@@ -27,7 +27,19 @@ <h1 class="page-header">Weather Data for Simulation</h1>
+            <li><a href="https://climate.onebuilding.org/">Climate.OneBuilding</a> - Climate.OneBuilding - climate data for more than 16,100 locations globally for building performance simulation. These include Typical Meteorological Years (TMY) published by a variety of organizations and new TMYx from Climate.OneBuilding. All data are available at no cost. For each location, the climate location zip file contains: 
+              <ul>
+				<li>EPW (EnergyPlus Weather Format)</li>
+				<li>CLM (ESP-r weather format)</li>
+				<li>WEA (Daysim weather format)</li>
+				<li>PVSyst (PV Solar weather design format)</li>
+				<li>DDY (ASHRAE 2021 Design Conditions or "file" design conditions in EnergyPlus format)</li>
+				<li>RAIN (hourly precipitation in m/hr, where available)</li>
+				<li>STAT (expanded EnergyPlus weather statistics)</li>
+              </ul>
+              <p>TMYx files are typical meteorological data derived from hourly weather data through 2021 in the ISD (US NOAA's Integrated Surface Database) using the TMY/ISO 15927-4:2005 methodologies. Currently, there are more than 16,100 TMYx locations in 249 countries available for free download. For the 2022/2023 release (this includes full TMYx, 2007-2021 TMYx but does not include the 2004-2018 TMYx), the solar data for each site has corresponding solar radiation from the ERA5 reanalysis data set. The ERA5 data, courtesy of Oikolab, provides a comprehensive, worldwide gridded solar radiation data set based on satellite data. The older 2004-2018 TMYx files use cloud cover and other variables to determine solar radiation.  Other data sets include: TMY3a (US and some others), TMY (US and some others), CWEC 2020 v2 (564 locations) and CWEC TDY 2020 (564 locations), AMTUes v2.5 (Uruguay), U.S. Climate Normals (1981-2010, 1991-2020, and 2006-2020 for 600 locations),  CTZ (California Climate Zones), CSWD (Chinese Standard Weather Data), BBSR (Germany), NIWA (New Zealand), RMY 2012 (Australia), IMS (Israel), ITMY (Iran), SWERA (multiple countries), INMET and TRY (Brazil), City University of Hong Kong, ISHRAE (India), IMGW (Poland), KISR (Kuwait), JGMY (Japan generated weather), TurTMY (Turkey), and ArgTMY (Argentina).</p>
+			  <br><br>
+			</li>
```

Comment:
Entirely new section for onebuilding, please advise if the raw or rendered formatting needs work.